.template/static_files_ignore.txt

```diff
@@ -1,6 +1,5 @@
 # Optional list of files which are actually static in the template
 # but are allowed to have different content in the current repository
-./devcontainer/Dockerfile
 
```

Comment:
```suggestion
./devcontainer/Dockerfile
```
I think you still need this in here.

Reply:
Sneaky little thing blended in so well with the comments that I missed it.
Added again in a786ffc5d304342a4a8d8da311ff0cfb883e489d